app/controllers/concerns/lol_connections.rb

```diff
@@ -1,127 +0,0 @@
-require 'lol'
```

Comment:
Since it appears that this file has been removed, the friendship functionality now fails because LolConnections is used in the Friendship controller. That will need to be fixed as part of this refactor.